### ISSUE TITLE
As a public user I want to see an image with a link to Open Forest on social media so that I can recognize the site.

### ISSUE THREAD

**Issue by ASprinkle** [USDAForestService/fs-open-forest-platform#1048]
## Notes
* What are things we should consider when making this story *

When Open Forest is linked on social media, an image is not associated with the web link.

## Acceptance Criteria
- [ ]

## Tasks
- [ ] 
- [ ] PO approved

## Definition of Done
- [ ] Pull requests meet technical definition of done
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by ASprinkle** [comment]
@lisaredux @lisamcmichael can you both help me test and refine this soon? There was an issue with this last year, and I do not think we ever addressed it in time for this year. Forests may be inclined to post more on social media if an image was automatically associated with a link to Open Forest weblink.

**Comment by lisamcmichael** [comment]
@lisaredux Let's grab some time to review this when you are ready. Thx!

**Comment by lisaredux** [comment]
@ASprinkle @lisamcmichael Definitely! I'll do a bit of research on what the links are grabbing for the  preview
